vars/npmRelease.groovy

```diff
@@ -6,27 +6,24 @@ def call(body) {
     body.delegate = config
     body()
 
-    def gitEmail = config.gitEmail ?: 'fabric8-admin@googlegroups.com'
-    def gitUserName = config.gitUserName ?: 'fabric8-release'
     def branch = config.branch
+    def flow = new io.fabric8.Fabric8Commands()
 
-    sh """
-       git config user.email ${gitEmail}
```

Comment:
email and username is getting hardcoded here?
if yes then `flow.setupGitSSH()` could take email id and username params

Reply:
I think so `username` and `email_id` are same everywhere hence this may not break anything

Reply:
@hrishin Its OK. We were not overriding it anywhere. Now things are at least consistent everywhere.